apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -217,6 +206,32 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 _consumer.Receive(new[] { entry });
             }
             catch (Exception) when (_loggerOptions.RetryOptions.ExceptionHandling == ExceptionHandling.Ignore) { }
+
+            Struct CreateStruct(IEnumerable<KeyValuePair<string, object>> fields)
+            {
+                Struct fieldsStruct = new Struct();
+                foreach (var pair in fields)
```

Comment:
This only prefixes with `_` if *all* chars are digits.
Why not prefix with `_` if just the first char is a digit?

Reply:
Hmm... good point, I think that would work - and be simpler, of course. Will fix. (I'll skip any empty fields at the same time, as I don't think they'd be valid.)

Reply:
Added one extra commit - PTAL. (It's very small :)